elements/pfe-band/src/pfe-band.scss

```diff
@@ -68,9 +55,8 @@ $LOCAL: band;
 
   // Wrapper sets padding and background visuals
   position: relative;
-  @extend %flex-fallback;
```

Comment:
Should we keep a flex fallback for older browsers just in case?

Reply:
The flex fallback didn't render well, it was auto filling the band pieces so header was beside the body. In my opinion it would be better if it stacked. 
![Scerenshot-flex-fallback](https://user-images.githubusercontent.com/20518754/67106240-7fb88e00-f1ca-11e9-974d-2b62df5ae684.png)

Reply:
👍

Reply:
Actually sorry, I was mistaken. The screenshot above is what it looks like if display:grid is rendering but no grid-areas are applied. The flex-fallback however still stacks everything and then breaks IE styles. If we feel this is important I could spend more time figuring out how to write the flex fallback? I'm thinking it would need to come in through the lightdom since browsers not rendering are grouping everything as 1 piece (that's my knowledge)? The we could wrap the flex-fallback to not target IE? 
<img width="1302" alt="Screenshot-flex-fallback-opera" src="https://user-images.githubusercontent.com/20518754/67107755-55b49b00-f1cd-11e9-9339-5c1048cba74c.png">

Is this worth the time to carve out more discovery on this for this story?

Reply:
No - we're good! 👍

Reply:
Gah wait, 🤦‍♂ I think I applied the fallback in the wrong place. Looking more into it now.